presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveWindowQueries.java

```diff
@@ -23,13 +27,28 @@ public TestHiveWindowQueries()
         super(true);
     }
 
+    protected List<String> getQueries(String rankingFunction)
+    {
+        List<String> queries = new ArrayList<String>();
```

Comment:
It seems like all the column projections are the same, and overclauses are different. Maybe write this as a loop that adds to queries list for each overClause to make it more readable.

Reply:
Thanks for the suggestion, the column projections are the same for 4 queries but the outer order by clauses are different.

Reply:
Changed to use a loop to iterate over the clauses, thanks!